samples/ImplicitFlow/AureliaOidcClientJs/src/login.ts

```diff
@@ -0,0 +1,65 @@
+import { autoinject } from "aurelia-framework";
+import { OpenId, User } from "./open-id/open-id";
+import { HttpClient } from "aurelia-fetch-client";
+
+@autoinject
+export class Login {
+
+    private authorizationServerMessage: string;
+    private resourceServerMessage: string;
+    private isLoggedIn: boolean = false;
+
+    constructor(private openId: OpenId, private httpClient: HttpClient) {
+        this.openId.UserManager.getUser().then((user: User) => {
+
+            console.log(user);
+            if (user === null || user === undefined) {
```

Comment:
Add a helper to the OpenId class that tests whether the user is logged in or not.